mirage-conduit.opam

```diff
@@ -8,13 +8,13 @@ tags:         "org:mirage"
 license:      "ISC"
 
 build: [
-  ["jbuilder" "subst" "-n" name] {pinned}
-  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["dune" "subst" "-n" name] {pinned}
```

Comment:
I'm not sure `dune subst` supports `-n`. Anyway it's safe to remove since it's in `dune-project`.